docker/launch_prebuilding.sh

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env bash
+set -euo pipefail
+
+# Parse input params
+python_version=${1:-lastest}
+cuda_version=${2:-11.8}
+gpu_backend=${3:-cuda}
+
+export FF_CUDA_ARCH=all
+export BUILD_LEGION_ONLY=ON
+
+# Build Docker Flexflow Container
+echo "building docker"
+./docker/build.sh flexflow
+export detached=ON
+./docker/run.sh flexflow
```

Comment:
since we don't need to run any additional command within the docker container after building it, we can avoid running the container